build.sh

```diff
@@ -43,6 +44,7 @@ BUILD_TYPE=Release
 INSTALL_TARGET=install
 CUDA_STATIC_RUNTIME=OFF
 PER_THREAD_DEFAULT_STREAM=OFF
+NO_CUDA_MALLOC_ASYNC_SUPPORT=OFF
```

Comment:
Double negative seems pretty confusing. Maybe do `CUDA_MALLOC_ASYNC_SUPPORT=ON` instead?

Reply:
AGREE=ON

Reply:
(Note same as DISAGREE=OFF but much clearer. :) )

Reply:
Good point, updated the build.sh and CMake not to use double negatives.